From 196901a192604e248e003ed84b353ce41bc6332b Mon Sep 17 00:00:00 2001
From: John Muir <muirj@google.com>
Date: Wed, 10 May 2017 16:14:24 -0700
Subject: [PATCH 05/13] Add vendor-specific message types for FM.

This change is for devices based on msm8998 hardware only. This
only impacts standalone VR devices based o that are using oc-dev.

Some message types used by QCOM are missing from
system/bt/hci/include/bt_vendor_lib.h.

Bug: 38055350
Test: BT compiles. Proprietary bits that depend on the
      definitions added by this commit now compile.
Change-Id: Ibe524528df6d86b7c0eb1f7df8d7074fcbcfbf9b
---
 libbt-vendor/include/bt_vendor_qcom.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/libbt-vendor/include/bt_vendor_qcom.h b/libbt-vendor/include/bt_vendor_qcom.h
index 12ba51e..b3eaa8c 100644
--- a/libbt-vendor/include/bt_vendor_qcom.h
+++ b/libbt-vendor/include/bt_vendor_qcom.h
@@ -43,6 +43,12 @@ typedef enum {
     BT_SOC_RESERVED
 }bt_soc_type;
 
+typedef enum {
+    FM_VND_OP_POWER_CTRL = (unsigned int)BT_VND_OP_A2DP_OFFLOAD_STOP + 1,
+    BT_VND_OP_FM_USERIAL_OPEN,
+    BT_VND_OP_FM_USERIAL_CLOSE,
+}bt_fm_serial;
+
 typedef enum {
     BT_VND_OP_ANT_USERIAL_OPEN = 254,
     BT_VND_OP_ANT_USERIAL_CLOSE
-- 
2.17.1

